cmd/k8s-pipeliner/main.go

```diff
@@ -42,6 +43,10 @@ func main() {
 					Name:  "timeout",
 					Usage: "override the default 72 hour timeout (unit: int)",
 				},
+				cli.StringSliceFlag{
+					Name:  "override",
+					Usage: "override an environment with a different environment (example --override=int-k8s:int), --override=<old env>:<new env>, must be separated by colon",
```

Comment:
Not 100% on the param options, but why stringslice as opposed to string? Should this be a comma-delimited list of pairs?

Reply:
--override=cat:dog --override=int:ops etc etc I'm open to either way, makes no difference to me

Reply:
your version would be: `--override=cat:dog,int:ops`

Reply:
makes sense. drop the commas in the usage and add (supports multiple entries)